### ISSUE TITLE
Group owner should be able to add/remove users

### ISSUE THREAD

**Issue by loraine-gueguen** [genouest/genouestaccountmanager#285]
It would be great if the group owner (required when creating a new group) could add and remove users from the group, as is the case with a project.

Probably related to https://github.com/genouest/genouestaccountmanager/issues/284

**Comment by osallou** [comment]
groups not managed by admins are linked to a project (though this may not be mandatory...) , so owner can add user to his project (and will be added to related group)

For use case where group is not related to a project, it could indeed be nice, but should we have groups not related to a project?
(@FanchTheSystem ?)

**Comment by loraine-gueguen** [comment]
> should we have groups not related to a project?

This case arises in ABiMS when handling private blast databanks. ping @lecorguille

**Comment by FanchTheSystem** [comment]
Yes I think at IFB too we use some group not related to project.

For example to give ssh access to some server.

**Comment by osallou** [comment]
yes, but those are "admin" limited groups, so only admins will anyway add/remove users. There is no need for a group owner to manipulate this

**Comment by loraine-gueguen** [comment]
> There is no need for a group owner to manipulate this

I think that it could be useful that one owner of a private blast databank to handle himself users that can have access to this bank, but admins can do that on request for sure.
From a support team member point of view :-)

**Comment by osallou** [comment]
but users request new groups to limit some bank/other access?

**Comment by loraine-gueguen** [comment]
Regarding bank only: yes, they ask for banks with restricted access. 
We can ask ourselves: can't we associate each private bank with a project and use the associated permissions? I'm not sure. For example, we need to give the blast application user (sequenceserver) read access to the banks so that the banks are available on the blast interface of Sequenceserver. However, this user is not relevant to have access to the whole project

I think we have other use cases with groups (mail aliases ?).

What do you think @lecorguille ?
